ihatemoney/templates/list_bills.html

```diff
@@ -1,5 +1,10 @@
 {% extends "sidebar_table_layout.html" %}
 
+{%- macro bill_amount(bill, currency=bill.original_currency, amount=bill.amount) %}
+    {# Undocumented currencyformat filter from flask_babel is forwarding to Babel format_currency #}
+    {{ amount|currencyformat(currency if currency != 'XXX' else '') }} ({{ _("%(amount)s each", amount=bill.pay_each_default(amount)|currencyformat(currency if currency != 'XXX' else '')) }})
+{% endmacro -%}
+
```

Comment:
Nice that you found out about this `currencyformat` filter!

I think it makes sense to write a small wrapper around it to handle the `XXX` currency, because that is something we want to do everywhere.

Also, small nitpick: with no currency, we still get non-breakable spaces for some locales:
```
In [1]: numbers.format_currency("42", "", locale="fr")
Out[1]: '42,00\xa0'
In [2]: numbers.format_currency("42", "", locale="no")                                                                                                       
Out[2]: '\xa042,00'
```
This looks slightly bad in the list of bills because there is too much space:
```
12,00  (6,00  each)
```

Reply:
It's a side-effect of currency formatting, since there is a non-breaking space before or after currency sign.
I crafted a new custom filter to handle this use case. I named it differently to avoid overwriting, but naming it was… difficult :)